apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -20,13 +20,34 @@ const alphabetMapper =  [
   commonMsg.answerOptionG(),
 ];
 
+const calculateNotAnswered = (multipleChoiceDataArr) => {
+    let total = 0;
+  multipleChoiceDataArr.forEach (studentsAnswersObj => {
+        if (studentsAnswersObj.percentAnswered) {
+            total += studentsAnswersObj.percentAnswered;
+        }
+    });
+
+    return (100 - total);
```

Comment:
I asked  "Do we have any multiple choice questions where students can select multiple answers, like a "select all that apply..." ?" in Slack because, if that's the case, you could run into a situation where `calculateNotAnswered` returns a negative number.  For example, let's say there are only two answer choices: "A" and "B" and students are allowed to select: only "A", only "B", none, or "A" AND "B".  If 100% of the students selected "A" AND "B", `total` in this function is 200, and you'd return -100. Let's confirm the requirements with Poorva.

Reply:
If students can select multiple answer options, I'm still concerned about `calculateNotAnswered` for this overview table. For example we could have a scenario like: 
![img_0564](https://user-images.githubusercontent.com/12300669/39642433-9ba82978-4f86-11e8-82bb-7a3d6f98931f.JPG)
In this case we would want the % not answered to be 25% because 1 out of 4 students did not answer the question; however the function as written would return 0%.

Reply:
This looks like a great unit test case!

Reply:
I concur!! Tests will be covered in the next PR.